### PR TITLE
ecryptfs 104 -> 106

### PR DESCRIPTION
**Pull request by obadz**

### CODE REVIEW

File: pkgs/tools/security/ecryptfs/default.nix [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-stdenv.mkDerivation {
-  name = "ecryptfs-104";
+stdenv.mkDerivation rec {
+  name = "ecryptfs-${version}";
+  version = "106";
 
   src = fetchurl {
-    url = http://launchpad.net/ecryptfs/trunk/104/+download/ecryptfs-utils_104.orig.tar.gz;
-    sha256 = "0f3lzpjw97vcdqzzgii03j3knd6pgwn1y0lpaaf46iidaiv0282a";
+    url = "http://launchpad.net/ecryptfs/trunk/104/+download/ecryptfs-utils_${version}.orig.tar.gz";
```
Comment (vbgl) [inline pkgs/tools/security/ecryptfs/default.nix]: The new URL looks wrong: should be `${version}` instead of `104`.

**Comment by obadz** [comment]
@vbgl, thanks for catching & sorry about that.

**Comment by nckx** [comment]
Needs a squashing; looks good otherwise.

**Comment by obadz** [comment]
Squashed.

**Comment by nckx** [comment]
Thanks! Merged.

**Pull request merged**